contribs/src/test/java/com/netflix/conductor/contribs/http/TestHttpTask.java

```diff
@@ -337,6 +339,35 @@ public void testOAuth() throws Exception {
 		
 		assertEquals("Task output: " + task.getOutputData(), Status.COMPLETED, task.getStatus());
 	}
+
+ 	@Test
+    public void testRequestHandled() throws Exception {
```

Comment:
Similar to my above comment of modifying their class vs extending and adding - that probably holds true here with the unit tests too.